### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by thongsav-usgs**
1) Limits expansion bug for multiple error
2) Upped testing coverage
3) Removed deprecated getTimeSeries function

### CODE REVIEW

File: R/utils-limits.R [hunk 1]
```diff
@@ -9,8 +9,17 @@ getErrorBarYLims <- function(error_bar_args){
   side <- ifelse(!is.null(error_bar_args[['side']]), error_bar_args[['side']], 2)
   
   ##If uncertainty does not exist, replace with 0.
-  ylow <- ifelse(!isEmptyOrBlank(error_bar_args[['y.low']]), abs(error_bar_args[['y.low']]), 0)
-  yhigh <- ifelse(!isEmptyOrBlank(error_bar_args[['y.high']]), abs(error_bar_args[['y.high']]), 0)
+  if(isEmptyOrBlank(error_bar_args[['y.low']])) {
+    ylow <- 0
+  } else {
+    ylow <- ifelse(lapply(error_bar_args[['y.low']], isEmptyOrBlank), 0, error_bar_args[['y.low']])
```
Comment (dpattermann-usgs) [inline R/utils-limits.R:15]: Why not just immediately run this (it will still catch the first case, right?)

Comment (thongsav-usgs) [reply R/utils-limits.R:15]: The first check tests if the variable is null. The second test if individual items inside the non-null list are null/empty.

Comment (thongsav-usgs) [reply R/utils-limits.R:15]: One of the tests breaks without the first check.

Comment (dpattermann-usgs) [reply R/utils-limits.R:15]: Ah, I see it now. It runs differently than I expected (it fills them in element by element, pretty useful). 👍

**Pull request merged**